types/api.go

```diff
@@ -5,6 +5,8 @@ import (
 	"fmt"
 	"reflect"
 	"strings"
+
+	tmbytes "github.com/tendermint/tendermint/libs/bytes"
```

Comment:
Same here... I really don't like to depend on tendermint for something as basic as our API types.